### PR TITLE
Added response messages for all relevant API's

### PR DESCRIPTION
**Pull request by bijob89**
## Description of the Change
Added response messages to enable users with better understanding of what is happening in the backend.

## Alternate Designs
UI needs to address the list of options provided to the user so that the server does not have to provide responses.

## Why Should This Be In Core?
Enables to understand the exact working instead of having a vague idea.

## Possible Drawbacks
Increased load on the server to check for different scenarios that arise.

### CODE REVIEW

File: mt2414/main.py [hunk 1]
```diff
@@ -319,7 +322,10 @@ def sources():
                         cursor.execute("INSERT INTO autogeneratedtokens (token, revision_num, source_id) VALUES (%s, %s, %s)", (t.decode("utf-8"), revision_num, source_id))
         cursor.close()
         connection.commit()
-        return '{"success":true, "message":"Existing source updated"}'
+        if changes:
```
Comment (faithfulmaster) [inline mt2414/main.py]: For both the above conditions, the changes array would always be appended. And the if condition would always be satisfied. Is it?

File: mt2414/main.py [hunk 2]
```diff
@@ -381,9 +387,14 @@ def autotokens():
...
-    source_id = cursor.fetchone()[0]
+    try:
+        source_id = cursor.fetchone()[0]
+    except:
+        return '{"success":false, "message":"Language and version combination not found"}'
     cursor.execute("SELECT token FROM autogeneratedtokens WHERE source_id = %s AND revision_num = %s", (source_id, revision))
     tokens = cursor.fetchall()
+    if not tokens:
+        return '{"success":false, "message":"Not a valid revision number"}'
     tr = {}
```
Comment (faithfulmaster) [inline mt2414/main.py]: Add a return statement with a valid statement when "success": true

File: mt2414/main.py [hunk 3]
```diff
@@ -404,18 +415,27 @@ def upload_tokens_translation():
...
+                cursor.execute("UPDATE autogeneratedtokens SET translated_token = %s, targetlang = %s WHERE token = %s AND source_id = %s AND revision_num = %s", (v, targetlang, k, source_id, revision))
+                changes.append(k)
+        cursor.close()
+        connection.commit()
+        if changes:
+            return '{"success":true, "message":"Token translations have been updated."}'
+        else:
+            return '{"success":false, "message":"No Changes. Token translations are already up-to-date."}'
+    else:
+        return '{"success":false, "message":"Upload tokens and translations"}'
```
Comment (faithfulmaster) [inline mt2414/main.py]: Check the message in the return statement. The message does not seem to convey the meaning correctly.

File: mt2414/main.py [hunk 4]
```diff
@@ -463,9 +486,13 @@ def get_concordance():
...
                 if ref_no not in db_book:
                     cursor.execute("INSERT INTO concordance (token, book_name, concordances, revision_num, source_id) VALUES (%s, %s, %s, %s, %s)", (token, ref_no, verse, revision_num, source_id))
+                    changes.append(book_name)
     cursor.close()
     connection.commit()
-    return '{"success":true, "message":"concordances created and stored in DB"}'
+    if changes:
+        return '{"success":true, "message":"concordances created and stored in DB"}'
+    else:
+        return '{"success":fasle, "message":"No changes made. Concordances are already up-to-date"}'
```
Comment (faithfulmaster) [inline mt2414/main.py]: Check spelling of false

File: mt2414/main.py [hunk 5]
```diff
@@ -514,7 +543,6 @@ def translations():
     for name, book in out:
         out_text_lines = []
         content = re.sub(r'([!"#$%&\'\(\)\*\+,-\.\/:;<=>\?\@\[\]^_`{|\}~। ])',r' \1 ', book)
-        print (content)
         for line in content.split("\n"):
             line_words = nltk.word_tokenize(line)
             new_line_words = []
```
Comment (faithfulmaster) [inline mt2414/main.py]: Return appropriate JSON messages for translations API also, as in some cases it is possible that the source text for a particular language and version is not available; or translation tokens for a particular target language is not available. And if available, return success message.

File: mt2414/main.py (hunk 2 above)
Comment (bijob89) [inline mt2414/main.py]: There is no need of a "success":true message as if the API is successful, it returns tokenwords as a json dump.

File: mt2414/main.py (hunk 1 above)
Comment (bijob89) [inline mt2414/main.py]: If there are no changes then nothing would be appended to the array. A value is appended into the array only if a change is made to the DB.

File: mt2414/main.py (hunk 5 above)
Comment (bijob89) [inline mt2414/main.py]: Done.

**Pull request merged**